leakpro/attacks/mia_attacks/abstract_mia.py

```diff
@@ -3,11 +3,14 @@
 from abc import ABC, abstractmethod
 
 import numpy as np
+import torch
```

Comment:
Previously we tried to avoid importing torch like this and just import modules we want. I think we still want that.

Reply:
Agreed! Removed this one and the whole memorization function from abstract_mia.py, to its own file!:)